services/user.py

```diff
@@ -0,0 +1,37 @@
+from db.models import User
+
+
+def create_user(*args, **kwargs) -> User:
+    user = User.objects.create_user(
+        *args, **kwargs
+    )
+    return user
+
+
+def get_user(user_id: int) -> User:
+    user = User.objects.get(id=user_id)
+    return user
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]